src/test_opencost_parquet_exporter.py

```diff
@@ -90,6 +90,27 @@ def test_get_config_no_window_end(self):
             self.assertNotIn('s3_bucket', config)
             self.assertEqual(config['params'][0][1], window)
 
+    def test_get_config_with_parquet_prefix_env_var(self):
+      """Test get_config overrides parquet prefix when env var is set."""
+      with patch.dict(os.environ, {
+        'OPENCOST_PARQUET_INCLUDE_IDLE': 'true',
+        'OPENCOST_PARQUET_IDLE_BY_NODE': 'true'}, clear=True):
+        config = get_config()
+        self.assertEqual(config['params'][2][0], 'includeIdle')
+        self.assertEqual(config['params'][2][1], 'true')
+
+        self.assertEqual(config['params'][3][0], 'idleByNode')
+        self.assertEqual(config['params'][3][1], 'true')
+
+    def test_get_config_with_idle_env_vars(self):
+      """Test get_config overrides parquet prefix when env var is set."""
+      with patch.dict(os.environ, {
```

Comment:
can you also add a test case to verify that passing as the prefix "{window_start_date.year}-{window_start_date.month}-{window_start_date.day}-{window_start_date.hour}" works ? 
this would prevent someone breaking your usecase in the future.

Reply:
Done ✅